.github/contributing.md

```diff
@@ -33,7 +33,7 @@ Hi! I'm really excited that you are interested in contributing to the SIX Webcom
 
 - Make sure tests pass!
 
-- Commit messages must follow the [commit message convention](./commit-convention.md) so that changelogs can be automatically generated. Commit messages are automatically validated before commit (by invoking [Git Hooks](https://git-scm.com/docs/githooks) via [simple-git-hooks](https://github.com/toplenboren/simple-git-hooks)).
+- Update the "upcoming" section inside _docs/changelog.md_ with the details of your changes.
```

Comment:
Maybe mention the changelog we follow, I think Iwan has mentioned that somewhere with a link.
@ibirrer maybe you can quickly post it here?

Reply:
Link to https://keepachangelog.com/en/1.1.0/